CONTRIBUTING.md

```diff
@@ -112,60 +112,69 @@ You only need to update the source files and Travis CI will take care of deployi
-
-If this is your first time generating changelog entries, add your personal access token with `public_repo` privileges to the `GITHUB_AUTH` environment variable by adding the following to your `.bashrc` file:
-
-```
-# Lerna Changelog Personal Access Token for shopify/slate repo
-
-export GITHUB_AUTH=your_personal_access_token
-```
-
-_Note: replace "your_personal_access_token" with your GitHub personal access token_
-
-Run the changelog generator:
-
-```
-yarn changelog
-```
-
-If nothing appears, you may not have any PRs tagged with appropriate labels in this release or you may have already published those changes. If you just released `v0.12.1` and the previous version was `v0.12.0`, run the following command to get the changes since `v0.12.0`:
-
-```
-yarn changelog -- --tag-from=v0.12.0
-```
-
-Copy the generated markdown from your terminal into [CHANGELOG.md](https://github.com/Shopify/slate/blob/master/CHANGELOG.md) and add any additional comments you wish to include. If the title of the autogenerated changelog is `Unreleased`, make sure you change it the new version name.
-
-Commit the changes directly to `master` branch, with a commit title of:
-
-```
-Changelog vX.X.X
-```
-
-_Note: replace "X.X.X" with new repo version number_
-
-Finally, paste the updates you made to the changelog in the release tag notes, see example: [v0.10.0 tag notes](https://github.com/Shopify/slate/releases/tag/v0.10.0).
-
 ## Publishing
 
-1.  Merge any changes you want to include in your next release into `master`.
-
-_Note: If you are merging multiple PRs into `master` with a single PR (e.g. you are merging a working branch called v0.11.0 with multiple fixes made from multiple PRs into `master`), then **do not squash and merge this PR** because you will loose valuable details in the auto generated changelog_
-
-2.  Update the [CHANGELOG.md](https://github.com/Shopify/slate/blob/master/CHANGELOG.md) as described above
-
-3.  To select a new version number, and publish packages to NPM, run:
-
-```
-yarn run publish
-```
-
-_Note: Make sure you are logged into your Shopify NPM account before publishing_
-
-```
-
-```
+⚠️ Note: You must have a Shopify Okta account ir order to login to Shipit and publish.
+
+1. Merge any changes you want to include in your next release into `master`.
+
+   _Note: If you are merging multiple PRs into `master` with a single PR (e.g. you are merging a working branch called v0.11.0 with multiple fixes made from multiple PRs into `master`), then **do not squash and merge this PR** because you will loose valuable details in the commit history_
+
+2. Checkout `master` and pull the latest from the origin
+
+   ```
+   slate $ git checkout master && git pull origin master
+   ```
+
+3. You can verify that there are indeed packages to be published (optional)
+
+   ```
+   slate $ yarn lerna updated
```

Comment:
Altho `lerna updated` still works, `lerna updated` is the official way. 

https://github.com/lerna/lerna/tree/master/commands/changed#readme